cypress/integration/seededFlows/loggedOutFlows/namespacedForemMobile.spec.js

```diff
@@ -0,0 +1,109 @@
+describe('Namespaced ForemMobile functions', () => {
+  function waitForMobileToBootstrap() {
+    cy.get('body').should('have.attr', 'data-runtime', 'ForemWebView-iOS');
+  }
+
+  // Make the runtime act as ForemWebView context
+  const runtimeStub = {
+    onBeforeLoad: (win) => {
+      Object.defineProperty(win.navigator, 'userAgent', {
+        value:
+          'Mozilla/5.0 (iPhone; CPU iPhone OS 14_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 ForemWebView/1.0',
+      });
+      Object.defineProperty(win.navigator, 'platform', { value: 'iPhone' });
+    },
+  };
+
+  describe('within ForemWebView context', () => {
+    describe('when logged in', () => {
+      beforeEach(() => {
+        cy.testSetup();
+        cy.fixture('users/adminUser.json').as('user');
+        cy.get('@user').then((user) => {
+          cy.loginUser(user).then(() => {
+            cy.visit('/', runtimeStub);
+            waitForMobileToBootstrap();
+          });
+        });
+      });
+
+      it('should load the ForemMobile namespaced functions', () => {
+        cy.window().then((win) => {
+          assert.isNumber(win.ForemMobile.retryDelayMs);
+          assert.isFunction(win.ForemMobile.getUserData);
+          assert.isFunction(win.ForemMobile.getInstanceMetadata);
+          assert.isFunction(win.ForemMobile.registerDeviceToken);
+          assert.isFunction(win.ForemMobile.unregisterDeviceToken);
+          assert.isFunction(win.ForemMobile.injectJSMessage);
+          assert.isFunction(win.ForemMobile.injectNativeMessage);
+          assert.isFunction(win.ForemMobile.userSessionBroadcast);
+        });
+      });
+
+      it('should return the instance metadata JSON when requested', () => {
+        cy.window().then((win) => {
+          var res = JSON.parse(win.ForemMobile.getInstanceMetadata());
+          assert.isObject(res);
+          assert.equal(res.domain, 'localhost:3000');
+          assert.isString(res.logo);
+          assert.isString(res.name);
+        });
+      });
+
+      it('should return user data when logged in', () => {
+        cy.window().then((win) => {
+          cy.fixture('users/adminUser.json').as('user');
+          cy.get('@user').then((expected) => {
+            const actual = JSON.parse(win.ForemMobile.getUserData());
+
+            expect(actual.username).to.equal(expected.username);
```

Comment:
Since we have the fixture data, it's perfect for assertion of the user data returned.